docs/en/Getting-Started.md

```diff
@@ -107,8 +107,7 @@ Alluxio configuration:
 $ ./bin/alluxio validateEnv local
 ```
 
-This will report potential problems that might prevent you from starting Alluxio services locally. If
-you configured Alluxio to run in a cluster and you want to validate environment on all nodes, you
+This will report potential problems that might prevent you from starting Alluxio services locally. If you configured Alluxio to run in a cluster and you want to validate environment on all nodes, you
```

Comment:
no real change here?

Reply:
I think it's just showing the overall difference from before the pull request to now. If you look at the first commit I added a sentence before and forgot to erase it

https://github.com/Alluxio/alluxio/pull/7818/commits/aca70968d78f1b655725391085f9e7c7aed8d959